lib/std/jsbigints.nim

```diff
@@ -209,3 +210,6 @@ runnableExamples:
     var b: JsBigInt
     doAssert b == big"0"
     doAssert b == JsBigInt.default
+  block `octal binary hexadecimal`:
```

Comment:
see https://github.com/timotheecour/Nim/issues/534#issuecomment-770108748

Reply:
Not using `break`.